editquality/utilities/autolabel.py

```diff
@@ -39,7 +40,10 @@
                              processing edits. [default: <cpu_count>]
     --input=<path>           The path to a file containing rev_ids to process
                              [default: <stdin>]
-    --output=<path>          The path to write labels to [default: <stdout>]
+    --output-needs-review=<path> The path to write observations in need of review.
+                                 [default: <stdout>]
+    --output-no-review=<path>    The path to write assumed good observations.
+                                 [default: <stdout>]
```

Comment:
This seems overly complicated.  I don't see why filtering after the fact is any more difficult and it's certainly less messy to allow the filter logic to be modular.